pkg/component/autoscaling/vpa/crd.go

```diff
@@ -71,3 +79,46 @@ func (v *vpaCRD) Destroy(ctx context.Context) error {
 
 	return nil
 }
+
+var (
+	// IntervalWaitForCRD is the interval used while waiting for the CRDs to become healthy
+	// or deleted.
+	IntervalWaitForCRD = 1 * time.Second
+	// TimeoutWaitForCRD is the timeout used while waiting for the CRDs to become healthy
+	// or deleted.
+	TimeoutWaitForCRD = 15 * time.Second
+	// Until is an alias for retry.Until. Exposed for tests.
+	Until = retry.Until
+)
```

Comment:
What's the benefit of these constants if there are no unit tests where they could get overridden?